Test/Tests/LogAdapterTests.swift

```diff
@@ -26,15 +26,15 @@ class LogAdapterTests: XCTestCase {
     }
 
     func test_logLevel() {
-        let underlyingLogger = ARTLog()
+        let underlyingLogger = Log()
```

Comment:
`AblyLogger`?

Reply:
fixed in https://github.com/ably/ably-cocoa/pull/1955/commits/ee87305959865a2daa9a6cc0dbfd91ebeb83a258